src/main/app/src/app/configuratie/configuratie.service.ts

```diff
@@ -111,4 +112,18 @@ export class ConfiguratieService {
     }
     return this.bpmnSupport$;
   }
+
+  readProtocolleringProvider(): Observable<string> {
+    if (!this.protocolleringProvider$) {
+      this.protocolleringProvider$ = this.http
+        .get(`${this.basepath}/brp/protocollering-provider`, {
+          responseType: "text",
+        })
+        .pipe(
```

Comment:
If this response should be cleaned, that should be done here. ie this logic should be added, before the value is returned for use in further logic, here:
```
  private getProtocolering(provider: string) {
    const clean = (provider ?? "")
      .toString()
      .replace(/['"\n\r\t]+/g, "")
      .trim();
    return clean === "iConnect";
  }
```